app/src/modules/settings/routes/flows/components/logs-sidebar-detail.vue

```diff
@@ -296,4 +307,9 @@ const steps = computed(() => {
 	color: var(--foreground-subdued);
 	font-style: italic;
 }
+
+.v-pagination {
+	justify-content: center;
+	margin-top: 24px;
```

Comment:
Should this be updated for a consistent look with the revisions sidebar?

```suggestion
	margin-top: 10px;
```

## Current 24px
<img width="210" alt="image" src="https://github.com/directus/directus/assets/26413686/82acb2f4-1e33-45d1-ba7a-813053ab08fe">

## 10px After
<img width="210" alt="image" src="https://github.com/directus/directus/assets/26413686/cb3bf423-6eda-4a93-9b4d-058272d9eb6d">

<img width="210" alt="image" src="https://github.com/directus/directus/assets/26413686/256f615a-d1a6-4dcb-8654-2df639c95633">

Reply:
I think the other way around! Make revisions also 24 instead of 10 👍🏻